runtime/wasm/src/module/mod.rs

```diff
@@ -535,6 +536,13 @@ impl<C: Blockchain> WasmInstance<C> {
             id,
             field
         );
+        link!(
+            "store.get_in_block",
+            store_get,
```

Comment:
This should be store_get_in_block, right?

Reply:
Ooh .. nice catch! Will fix that